cmd/repo/setup/runner.go

```diff
@@ -67,6 +68,14 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		return microerror.Mask(err)
 	}
 
+	ChecksFilterRegexp, err := regexp.Compile(r.flag.ChecksFilter)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	if r.flag.ChecksFilter == "" {
+		ChecksFilterRegexp = nil
+	}
+
```

Comment:
I would prefer this, so we do not even bother compiling the regexp when we ignore the filter.

```suggestion
	var ChecksFilterRegexp *regexp.Regexp
	if r.flag.ChecksFilter != "" {
		ChecksFilterRegexp, err := regexp.Compile(r.flag.ChecksFilter)
		if err != nil {
			return microerror.Mask(err)
		}
	}
```